### PR TITLE
finish refactor

### PR DESCRIPTION
**Pull request by linc-mvla**
refactored folders
it builds, but haven't tested
rework SwerveModule and SwerveDrive to look prettier

### CODE REVIEW

File: src/main/cpp/Robot.cpp [hunk 1]
```diff
@@ -14,7 +14,7 @@
 #include <fmt/core.h>
 #include <frc/smartdashboard/SmartDashboard.h>
 
-#include "Constants.h"
+#include <Drive/DriveConstants.h>
```
Comment (jonathanhliu21) [inline src/main/cpp/Robot.cpp]: Make this quotation include rather than angle brackets

File: src/main/include/Climb/Climb.md
Comment (jonathanhliu21) [inline src/main/include/Climb/Climb.md:1]: Dont' include MD files. Instead, have an explanation of the code in a Doxygen comment or on the README md.

File: src/main/include/Drive/DriveConstants.h [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
- */
-
 #pragma once
 
 #include <cstddef>
 
-const double NEAR_ZERO_TOLERANCE = 0.000001;
+/**
+ * A file of constants for the drivebase
+ */
```
Comment (jonathanhliu21) [inline src/main/include/Drive/DriveConstants.h:4]: Put this above pragma once statement

File: src/main/include/GeneralConstants.h [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/**
+ * A file of constants
```
Comment (jonathanhliu21) [inline src/main/include/GeneralConstants.h]: Change to "A file of *general* constants"

File: src/main/include/Util/UtilConstants.h [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+/**
```
Comment (jonathanhliu21) [inline src/main/include/Util/UtilConstants.h:2]: Put this above pragma statement.

**Review by jonathanhliu21 — CHANGES_REQUESTED** [review]
small changes, not much besides deleting md files

File: src/main/cpp/Robot.cpp (hunk 1 above)
Comment (linc-mvla) [reply src/main/cpp/Robot.cpp]: Isn't the folder in the local directory?

Comment (jonathanhliu21) [reply src/main/cpp/Robot.cpp]: Yes, so quotations are preferred.

**Review by jonathanhliu21 — APPROVED** [review]
lgtm

**Pull request merged**